packages/office-ui-fabric-react/src/Utilities.ts

```diff
@@ -1,2 +1,3 @@
 import './version';
 export * from '@uifabric/utilities';
+export * from '@fluentui/react-window-provider';
```

Comment:
Wondering if we should make a separate file for this since it's not actually in utilities?

Reply:
thought about it more - I will move to `lib/WindowProvider`